### ISSUE TITLE
Is there any plan to support dynamic ip PPPOE?

### ISSUE THREAD

**Issue by thinhdinh25** [alexkirsz/dispatch#21]
I'm having 3 pppoe and getting trouble in loading balance them. Then i found this project but it must provide correct IP, which is very hard for me to do because IP changes each reboot. So is there any plan to support them?

**Comment by ForgetFullness** [comment]
@alexkirsz yes it would be fixed with interface naming, but i couldn't find anything about using the inteface name insted of ip, make sure you update the readme aswell.
so if i simply use "Ethernet" insted of "1.1.1.1" it would work?
